scripts/install_prerequisites.sh

```diff
@@ -34,11 +34,23 @@ sudo apt-get install python3-dev
 sudo apt-get install python3-pip
 sudo apt-get install unzip
 sudo apt-get install python3-yaml
-# This is only done to address an
-#     "ImportError: No module named functools_lru_cache"
-# error. See the Troubleshooting page for details:
-#    https://github.com/oppia/oppia/wiki/Troubleshooting
```

Comment:
Is this info still relvent?

Reply:
No it is not, this library is native in py3.